recipes/metplus/meta.yaml

```diff
@@ -0,0 +1,192 @@
+    folder: atlas
+
+build:
+  skip: True  # [win]
+  number: 0
+  missing_dso_whitelist:
+    # eckit/atlas libraries are installed within this recipe
+    {% if osx %}
+    - libeckit.dylib
+    - libeckit_geometry.dylib
+    - libatlas.dylib
+    {% else %}
+    - libeckit.so
+    - libeckit_geometry.so
+    - libatlas.so
+    {% endif %}
+  entry_points:
+    - run_metplus.py = metplus.scripts.run_metplus:cli_main
+
+# Notes:
+# - GCC 12 on Linux and fftw are needed to install atlas
+# - libopenblas v0.3.28 is needed using 0.3.29 causes eckit to not find
+#  the library and it compiles with other libraries that are not included
+#  in this recipe.
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib("c") }}
+    - {{ compiler('fortran') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - libcxx
+    - cairo
+    - freetype
+    - gsl
+    - libnetcdf
+    - netcdf-cxx4
+    - libopenblas 0.3.28
+    - python 3.10
+    - zlib
+    - nceplibs-bufr
+    - python-configuration
+    - jasper
+    - nceplibs-g2c
+    - hdf4
+    - hdfeos2
+    - proj
+    - fftw
+    - _openmp_mutex 4.5  # [linux]
+    - llvm-openmp  # [osx]
+
+  host:
+    - python 3.10
+    - pip
+    - gsl
+    - netcdf-cxx4
+    - zlib
+    - cairo
+    - freetype
+    - nceplibs-bufr
+    - nceplibs-g2c
+    - xarray
+    - numpy
+    - pandas
+    - pyyaml
+    - scipy
+    - fftw
+    - ncurses
+    - hdfeos2
+    - _openmp_mutex  # [linux]
+    - llvm-openmp  # [osx]
+    - libcxx
+    - setuptools
+    - setuptools-git-versioning >=2.0,<3
+
+  run:
+    - gsl
+    - netcdf-cxx4
+    - libopenblas
+    - python 3.10
+    - nceplibs-bufr
+    - nceplibs-g2c
+    - hdf4
+    - ncurses
+    - bzip2
+    - lz4-c
+    - snappy
+    - libaec
+    - libcurl
+    - fftw
+    - xarray
+    - numpy >=1.24,<2
+    - pandas
+    - pyyaml
+    - netcdf4
+    - scipy
+    - _openmp_mutex  # [linux]
+    - llvm-openmp  # [osx]
+
+test:
+  commands:
+    - test -f $PREFIX/bin/ascii2nc
+    - test -f $PREFIX/bin/ensemble_stat
+    - test -f $PREFIX/bin/gen_ens_prod
+    - test -f $PREFIX/bin/gen_vx_mask
+    - test -f $PREFIX/bin/grid_diag
+    - test -f $PREFIX/bin/grid_stat
+    - test -f $PREFIX/bin/ioda2nc
+    - test -f $PREFIX/bin/lidar2nc
+    - test -f $PREFIX/bin/madis2nc
+    - test -f $PREFIX/bin/mode
+    - test -f $PREFIX/bin/mode_analysis
+    - test -f $PREFIX/bin/mtd
+    - test -f $PREFIX/bin/pb2nc
+    - test -f $PREFIX/bin/pcp_combine
+    - test -f $PREFIX/bin/plot_data_plane
+    - test -f $PREFIX/bin/plot_point_obs
+    - test -f $PREFIX/bin/point2grid
+    - test -f $PREFIX/bin/point_stat
+    - test -f $PREFIX/bin/regrid_data_plane
+    - test -f $PREFIX/bin/series_analysis
+    - test -f $PREFIX/bin/stat_analysis
+    - test -f $PREFIX/bin/tc_diag
+    - test -f $PREFIX/bin/tc_gen
+    - test -f $PREFIX/bin/tc_pairs
+    - test -f $PREFIX/bin/tc_rmw
+    - test -f $PREFIX/bin/tc_stat
+    - test -f $PREFIX/bin/wavelet_stat
```

Comment:
Do any of these tools print help or version messages? If so, you should just call them directly instead of just checking they exist in order to check for broken links, missing symbols, etc.